src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -2,32 +2,43 @@
 
 public class DefaultCountingOutRhymer {
 
-	private int[] numers = new int[12];
+    private static final int minusOne = -1;
+    private static final int eleven = 11;

```

Comment:
This constant is related to "twelve" (wrong names!). Changing only one leads to an error (inconsistency).